### PR TITLE
[loss_op]Denote the returned value is already mean, it happens quite a few tim…

### PR DESCRIPTION
**Pull request by zuoxingdong**
…es that we often add tf.reduce_mean after that since tf.nn.sparse... returns loss without averaging it.

### CODE REVIEW

**Comment by martinwicke** [comment]
Can you make this change to python/ops/losses/loss.py as well? And for both the sparse and non-sparse versions?

**Comment by zuoxingdong** [comment]
@martinwicke Since I didn't manage to find python/ops/losses/losses.py in the patch for this pull request, so I created a new patch with a new request #6359 for this additional change.

**Comment by martinwicke** [comment]
Ok, I'll close this one then.

**Comment by martinwicke** [comment]
Oh wait, I see. Can you add the content of this PR to the other one? It'll be cleaner that way.
